seth/client_builder.go

```diff
@@ -227,6 +227,13 @@ func (c *ClientBuilder) WithArtifactsFolder(folder string) *ClientBuilder {
 	return c
 }
 
+// WithGethWrappersFolders sets list of folders where the Geth wrappers are stored. Seth will load ABIs from all wrappers it finds in theses folders (including subfolders).
+// Default value is an empty string (= loading disabled).
+func (c *ClientBuilder) WithGethWrappersFolders(folders []string) *ClientBuilder {
```

Comment:
Let's add it to our docs? https://github.com/smartcontractkit/chainlink-testing-framework/blob/tt_1511_load_abi_gethwrappers/seth/README.md